### PR TITLE
Adding updates for SDS jenkins config

### PR DESCRIPTION
**Pull request by endakelly**

### CODE REVIEW

**Review by timja — COMMENTED** [review]
can you please attach a full page screenshot?

**Comment by endakelly** [comment]
image

File: source/onboarding/team/index.html.md.erb [hunk 1]
```diff
@@ -79,17 +79,38 @@ which can easily be hit if names are too long.
 
 ## Jenkins org setup
 
+There are Jenkins servers setup for each business area within Platform Operations: CFT and SDS
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:82]: Platform Operations is not a business area

```suggestion
There are Jenkins servers for: CFT (Civil, Family & Tribunals) and SDS (Shared Digital Services)
```

File: source/onboarding/team/index.html.md.erb [hunk 2]
```diff
@@ -79,17 +79,38 @@ which can easily be hit if names are too long.
...
+
+CFT stands for Civil, Family & Tribunal
+
+SDS stands for Shared Services
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:86]:
```suggestion
```

File: source/onboarding/team/index.html.md.erb [hunk 3]
```diff
@@ -79,17 +79,38 @@ which can easily be hit if names are too long.
...
     TEAM_NAME=$1
     az keyvault secret set -n  hmcts-jenkins-${TEAM_NAME}-ghapp -f converted-github-app.pem --vault-name cftptl-intsvc
     az keyvault secret set -n  hmcts-jenkins-${TEAM_NAME}-ghapp -f converted-github-app.pem --vault-name cftsbox-intsvc
-    ``` 
- 4. Add your organisation config in [cnp-jenkins-config](https://github.com/hmcts/cnp-jenkins-config/pull/402)
- 5. Add a Jenkins build dashboard in [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/7221)
+    ```
+ 4. Add your organisation config.
+
+        CFT: [cnp-jenkins-config](https://github.com/hmcts/cnp-jenkins-config/pull/402)
```
Comment (timja) [inline source/onboarding/team/index.html.md.erb:106]: this screenshot you provided shows that this is not rendering properly

**Review by timja — CHANGES_REQUESTED** [review]

**Comment by endakelly** [comment]
image

**Review by msl8r — APPROVED** [review]
LGTM (I think) :D

**Review by timja — APPROVED** [review]
let's see how it goes

**Pull request merged**